blockapps-strato/azuredeploy.json

```diff
@@ -0,0 +1,231 @@
+            "dependsOn": [
+                "[variables('storageAccountPath')]",
+                "[variables('networkInterfacePath')]"
+            ],
+            "properties": {
+                "provisioningState": 0,
+                "diagnosticsProfile": {
+                    "bootDiagnostics": {
+                        "enabled": false
+                    }
+                },
+                "networkProfile": {
+                    "networkInterfaces": [
+                        {
+                            "id": "[variables('networkInterfaceId')]"
+                        }
+                    ]
+                },
+                "osProfile": {
+                    "computerName": "[variables('name')]",
+                    "adminUsername": "[variables('username')]",
+                    "adminPassword": "[variables('password')]",
+                    "linuxConfiguration": {
+                        "disablePasswordAuthentication": "false"
+                    }
+                },
+                "storageProfile": {
+                    "osDisk": {
+                        "caching": "ReadWrite",
+                        "vhd": {
+                            "uri": "[variables('vhdURI')]"
+                        },
+                        "image": {
+                            "uri": "[variables('imageURI')]"
+                        },
+                        "createOption": "FromImage",
+                        "name" : "[concat(variables('name'), 'osDisk')]",
+                        "osType": "Linux"
+                    }
+                },
+                "hardwareProfile": {
+                    "vmSize": "[variables('vmSize')]"
+                }
+            },
+            "apiVersion": "2015-06-15"
+        }
+    ],
+    "parameters": {
+        "instanceName": {
+            "type": "string",
+            "metadata": {
+                "description" : "What to call this virtual machine"
+            }
+        },
+        "blockchain": {
+            "type": "string",
+            "allowedValues" : ["live", "dev"],
+            "metadata": {
+                "description" : "Which STRATO configuration to enable: 'live' is the Ethereum network; 'dev' is a private developer blockchain"
+            }
+        },
+        "password" : {
+            "type" : "secureString",
+            "metadata" : {
+                "description" : "Password for admin account 'strato'"
+            }
+        }
+    },
+    "variables": {
+        "imageVHDs": {
+            "live" : "strato-live-image-osDisk.3dfba3b1-de95-49d4-8aaa-ea37a17f5eaa.vhd",
+            "dev"  : "strato-dev-image-osDisk.a136c286-678b-48b9-86b9-56052a28d849.vhd"
+        },
+        "imageVHD": "[variables('imageVHDs')[parameters('blockchain')]]",
+        "imageURI": "[concat('https:\/\/', variables('imageStorage'), '.blob.core.windows.net\/system\/Microsoft.Compute/Images/vhds/', variables('imageVHD'))]",
```

Comment:
Yes, this is our storage account. It's the reason for the INTERNAL USE ONLY
tag on top.  Is that okay for now?

On Fri, Nov 6, 2015 at 5:41 PM, Kay Singh <notifications@github.com> wrote:

> In blockapps-strato/azuredeploy.json
> <https://github.com/Azure/azure-quickstart-templates/pull/816#discussion_r44195948>
> :
>
> > +            }
> > +        },
> > +        "password" : {
> > +            "type" : "secureString",
> > +            "metadata" : {
> > +                "description" : "Password for admin account 'strato'"
> > +            }
> > +        }
> > +    },
> > +    "variables": {
> > +        "imageVHDs": {
> > +            "live" : "strato-live-image-osDisk.3dfba3b1-de95-49d4-8aaa-ea37a17f5eaa.vhd",
> > +            "dev"  : "strato-dev-image-osDisk.a136c286-678b-48b9-86b9-56052a28d849.vhd"
> > +        },
> > +        "imageVHD": "[variables('imageVHDs')[parameters('blockchain')]]",
> > +        "imageURI": "[concat('https:\/\/', variables('imageStorage'), '.blob.core.windows.net\/system\/Microsoft.Compute/Images/vhds/', variables('imageVHD'))]",
>
> I'm assuming this is a storage account you own and only you'll be able to
> deploy this and not customers?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-quickstart-templates/pull/816/files#r44195948>
> .
>